content-src/components/DiscoveryStreamBase/DiscoveryStreamBase.jsx

```diff
@@ -111,6 +111,10 @@ export class _DiscoveryStreamBase extends React.PureComponent {
   }
 
   renderComponent(component, embedWidth) {
+    const ENGAGEMENT_LABEL_ENABLED = this.props.Prefs.values[
+      `discoverystream.engagementLabelEnabled`
+    ];
+
```

Comment:
Sorry for not clarifying earlier, what I meant was moving pref name outside [near line 27](https://github.com/mozilla/activity-stream/pull/5331/files#diff-e1ad7b3efdc1ff037a2adb747c057066R27) similar to [endpoints pref](https://searchfox.org/mozilla-central/source/browser/components/newtab/lib/DiscoveryStreamFeed.jsm#56) and pass value to attribute display_engagement_labels in L220 as you were doing before as this.props.Prefs.values[PREF_ENGAGEMENT]